cpp/src/xs.h

```diff
@@ -268,7 +289,7 @@ namespace CRXS {
          *
          *  \return double XS             Cross section in mbarn/GeV^2
          */
-        static double inv_AA_Dbar_CM( double s, double xF_Dbar, double pT_Dbar, int A_projectile, int N_projectile, int A_target, int N_target, int parametrization, int coalescence);
+        static double inv_AA_Dbar_CM( double s, double xF_Dbar, double pT_Dbar, int A_projectile, int N_projectile, int A_target, int N_target, int parametrization, int coalescence, double p0_val);
```

Comment:
Here it is important to put a default value for p0_val. If you don't do that you loose a backward compatibility of the software. So, it should be: 
double p0_val=0.190